e2e/wdio/platform/fixtures/testData/date-time-picker.ts

```diff
@@ -1,8 +1,6 @@
-export default
-{
-    date: '1/1/2021, 1:01 PM',
-    highlightedColor: ['rgb(8,84,160)', 'rgba(8,84,160,1)'],
-    compactDate: '10/10/2018, 9:35 PM',
-    year2030: 2030,
-    currentDate: `${new Date().getMonth() + 1}/${new Date().getDate()}/${new Date().getFullYear()}, 1:01 PM`
-}
+export const date = '1/1/2021, 1:01 PM';
+export const highlightedColor = ['rgb(8,84,160)', 'rgba(8,84,160,1)'];
+export const compactDate = '10/10/2018, 9:35 PM';
+export const year2030 = 2030;
+export const currentDate = `${new Date().getMonth() + 1}/${new Date().getDate()}/${new Date().getFullYear()}, 1:01 PM`;
+export const text = 'test';
```

Comment:
if you want to avois writing 'export const' so many times,, you can do it like:

export const
date = '1/1/2021, 1:01 PM',
highlightedColor = ['rgb(8,84,160)', 'rgba(8,84,160,1)'],
compactDate = '10/10/2018, 9:35 PM',
year2030 = 2030,
currentDate = `${new Date().getMonth() + 1}/${new Date().getDate()}/${new Date().getFullYear()}, 1:01 PM`,
text = 'test';

Reply:
you should not use this approach

Reply:
@valorkin could you explain why? I know it's not recommended but I never found online a good explanation :D

Reply:
I suppose it will be hard to find one, because roots are in es3 or es5 and `var`s. It's more about style and readability.
Rather more appropriate approach is (IMHO)
```ts
const compactDate = '10/10/2018, 9:35 PM';
const year2030 = 2030;
const currentDate = `${new Date().getMonth() + 1}/${new Date().getDate()}/${new Date().getFullYear()}, 1:01 PM`;
const text = 'test';

export {compactDate, year2030, currentDate, text};
```